src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1761,30 +1761,21 @@ public void loadGestures() {
    * @param directory
    *          - the directory that contains the gesture python files.
    */
-  public void loadGestures(String directory) {
-    // TODO: iterate over each of the python files in the directory
+  public boolean loadGestures(String directory) {
```

Comment:
If you're changing the return type for this, you should update the javadoc comment to explain what the return value is and means.  If it's a boolean, when is it true and when is it false.

Reply:
reverted